Tiltfile

```diff
@@ -107,7 +93,7 @@ else:
     context='.',
     entrypoint="/cluster-agent/cluster-agent -c /etc/kubetail/config.yaml",
     only=[
-      './crates/rgkl',
```

Comment:
`'./.tilt/cluster-agent'` should be sufficient in this list (no need for `'./proto'` either) because the container only needs the final executable to run.